plugins/module_utils/cmd_runner.py

```diff
@@ -204,12 +205,17 @@ def __init__(self, module, command, arg_formats=None, default_args_order=(),
             environ_update = {}
         self.environ_update = environ_update
 
-        self.command[0] = module.get_bin_path(self.command[0], opt_dirs=path_prefix, required=True)
+        _cmd = self.command[0]
+        self.command[0] = _cmd if os.path.isabs(_cmd) else module.get_bin_path(_cmd, opt_dirs=path_prefix, required=True)
```

Comment:
How about also allowing relative paths?
```suggestion
        self.command[0] = _cmd if (os.path.isabs(_cmd) or '/' in _cmd) else module.get_bin_path(_cmd, opt_dirs=path_prefix, required=True)
```
On the other hand, might be better to handle that in another PR, as this seems a bit hacky to me.

Reply:
Nah, let's do it